include/sys/dsl_scan.h

```diff
@@ -70,6 +71,7 @@ typedef struct dsl_scan_phys {
 
 typedef enum dsl_scan_flags {
 	DSF_VISIT_DS_AGAIN = 1<<0,
+	DSF_SCRUB_PAUSED = 1<<1,
```

Comment:
I can just do a full upstream of scrub-pause to OpenZFS. Unless we want soak time in ZoL first?